src/da65.vcxproj

```diff
@@ -86,6 +86,7 @@
     <ClCompile Include="da65\infofile.c" />
     <ClCompile Include="da65\labels.c" />
     <ClCompile Include="da65\main.c" />
+    <ClCompile Include="da65\opc4510.c" />
```

Comment:
I guess this change isn't actually related to this pull request but rather a fix/addon to a 3rd party change. I generally prefer a separate pull request in such a case but I'm willing to merge this here.